railties/lib/rails/generators/rails/app/templates/config/deploy.yml.tt

```diff
@@ -30,7 +30,7 @@ env:
     - RAILS_MASTER_KEY
   clear:
     # Set this to the number of cores you wish the application to use on each server.
-    # If not set, app will attempt to use all available cores (but may guess wrong on some cloud hosts!)
+    # You can use "auto" and the app will attempt to use all available cores (but may guess wrong on some cloud hosts!)
     WEB_CONCURRENCY: 1
```

Comment:
With this PR, since `WEB_CONCURRENCY` will be 1 by default, I wonder if:

- the line above should be commented out by default, maybe with `# WEB_CONCURRENCY: auto` or a different value.
-  the lines above should be removed altogether. If people are interested in tweaking worker counts, they will have read `config/puma.rb` anyways, and if they so choose will edit their config/deploy.yml accordingly.

Reply:
Good point. Do you want to open a PR?

Reply:
Sure, I opened 2 PRs so since I wasn't sure which bullet point we wanted to go with:

- https://github.com/rails/rails/pull/52536
- https://github.com/rails/rails/pull/52537